prediction_market_agent/agents/replicate_to_omen_agent/deploy.py

```diff
@@ -26,7 +26,7 @@ class ReplicateSettings(BaseSettings):
 
     N_TO_REPLICATE: int
     INITIAL_FUNDS: str
-    CLOSE_TIME_UP_TO_N_DAYS: int
+    CLOSE_TIME_UP_TO_N_DAYS: list[int]
```

Comment:
So we can force to replicate markets that will close soon, but at the same time these that will close in a year or so